src/lib/swapper/swappers/OneInchSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -24,88 +25,89 @@ export async function getTradeQuote(
     chainId,
     sellAsset,
     buyAsset,
-    sellAmountBeforeFeesCryptoBaseUnit,
     accountNumber,
     affiliateBps,
+    eip1559Support,
+    receiveAddress,
   } = input
+  const sellAmount = input.sellAmountBeforeFeesCryptoBaseUnit
 
-  if (sellAmountBeforeFeesCryptoBaseUnit === '0') {
```

Comment:
Are we now missing that check?

Reply:
Replaced with 
![image](https://github.com/shapeshift/web/assets/35275952/cd1608f7-0731-4ff2-9429-fd72b3a950e3)